style.css

```diff
@@ -1,7 +1,37 @@
+
 .header{
 	font-size: 36px;
 	letter-spacing: 1.2px;
 	font-family: helvetica;
 	color:#536240;
 	font-weight: 700;
+}
+.info{
+	position: relative;
+	display: flex;
+	left: 800px;
+	bottom:90px;
```

Comment:
Also bottom 90px, it's a bit to high, please try to rewrite this styling to have the same output, but with a better styling :) 
https://css-tricks.com/snippets/css/a-guide-to-flexbox/ check this link, add it as a bookmark to your browser, you are going to need it ;)